wren-ai-service/src/__main__.py

```diff
@@ -100,6 +100,7 @@ async def exception_handler(request, exc: Exception):
 
 @app.exception_handler(RequestValidationError)
 async def request_exception_handler(request, exc: Exception):
+    print(str(exc))
```

Comment:
I think we could remove printing the exception here, FastAPI handle the exception and also leave the log in console